cmd/lncli/main.go

```diff
@@ -252,6 +264,21 @@ func main() {
 			Name:  "macaroonip",
 			Usage: "if set, lock macaroon to specific IP address",
 		},
+		cli.StringFlag{
+			Name: "profile, p",
+			Usage: "instead of reading settings from command " +
+				"line parameters or using the default " +
+				"profile, use a specific profile instead. if " +
+				"a default profile is set, this flag can be " +
+				"set to an empty string to disable reading " +
+				"values from the profiles file",
+		},
+		cli.StringFlag{
+			Name: "macfromjar",
+			Usage: "use this macaroon from the profile's " +
+				"macaroon jar instead of the default one. " +
+				"can only be used if profiles are defined",
```

Comment:
Is there a reason to not capitalize sentence starts?

Reply:
Only because all the other flags don't have any punctuation or capitalization. But I changed everything to be proper sentences.